### PR TITLE
Update release checklist: homebrew and solc-js instructions

### PR DESCRIPTION
**Pull request by nikola-matic**

### CODE REVIEW

File: ReleaseChecklist.md [hunk 1]
```diff
@@ -76,7 +76,21 @@ At least a day before the release:
  - [ ] Create a pull request in solc-bin and merge.
 
 ### Homebrew and MacOS
- - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/solidity.rb).
+ - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solidity.rb).
```
Comment (r0qs) [inline ReleaseChecklist.md]: Actually, maybe we could remove this line.

Comment (nikola-matic) [reply ReleaseChecklist.md]: Sure, it's described in the manual update step anyway.

File: ReleaseChecklist.md [hunk 2]
```diff
@@ -76,7 +76,20 @@ At least a day before the release:
  - [ ] Create a pull request in solc-bin and merge.
 
 ### Homebrew and MacOS
- - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/solidity.rb).
+ - [ ] Create a local fork of the repository at ``git@github.com:Homebrew/homebrew-core.git``
+ - [ ] Make sure that ``brew`` is installed on your system, or use the official Homebrew docker container:
+ ``docker run --interactive --tty --rm --pull always --volume "$(pwd):/homebrew-core" --workdir "/homebrew-core" homebrew/ubuntu22.04:latest /bin/bash``
+#### Manual Update
+ - [ ] Open the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solidity.rb) using your preferred text editor and update the following fields:
+   - url: Update the URL to reflect the new source file for the updated version.
```
Comment (cameel) [inline ReleaseChecklist.md:84]: Spacing and indentation is off here:

```suggestion
 - [ ] Make sure that ``brew`` is installed on your system, or use the official Homebrew docker container:
     ``docker run --interactive --tty --rm --pull always --volume "$(pwd):/homebrew-core" --workdir "/homebrew-core" homebrew/ubuntu22.04:latest /bin/bash``

#### Manual Update
 - [ ] Open the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solidity.rb) using your preferred text editor and update the following fields:
     - url: Update the URL to reflect the new source file for the updated version.
```

File: ReleaseChecklist.md [hunk 3]
```diff
@@ -76,7 +76,20 @@ At least a day before the release:
  - [ ] Create a pull request in solc-bin and merge.
 
 ### Homebrew and MacOS
- - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/solidity.rb).
+ - [ ] Create a local fork of the repository at ``git@github.com:Homebrew/homebrew-core.git``
+ - [ ] Make sure that ``brew`` is installed on your system, or use the official Homebrew docker container:
+ ``docker run --interactive --tty --rm --pull always --volume "$(pwd):/homebrew-core" --workdir "/homebrew-core" homebrew/ubuntu22.04:latest /bin/bash``
```
Comment (cameel) [inline ReleaseChecklist.md:81]:
```suggestion
 ``docker run --interactive --tty --rm --pull always --volume "$PWD:/homebrew-core" --workdir /homebrew-core homebrew/ubuntu22.04``
```

File: ReleaseChecklist.md [hunk 4]
```diff
@@ -76,7 +76,20 @@ At least a day before the release:
...
+#### Manual Update
+ - [ ] Open the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solidity.rb) using your preferred text editor and update the following fields:
+   - url: Update the URL to reflect the new source file for the updated version.
+   - sha256: Replace with the new SHA256 checksum of the source file. You can obtain it using a command like:
```
Comment (cameel) [inline ReleaseChecklist.md:85]:
```suggestion
   - ``url``: Update the URL to reflect the new source file for the updated version.
   - ``sha256``: Replace with the new SHA256 checksum of the source file. You can obtain it using a command like:
```

File: ReleaseChecklist.md [hunk 5]
```diff
@@ -76,7 +76,20 @@ At least a day before the release:
...
+     ``curl -L https://github.com/ethereum/solidity/releases/download/v$VERSION/solidity_$VERSION.tar.gz | sha256sum``
+ - [ ] Test your changes locally by running the following command from the root directory of your fork: ``brew install --build-from-source ./Formula/s/solidity.rb``
+ - [ ] Open a PR in the Homebrew repository.
+#### Automatic Update
+ - [ ] Create a GitHub token and run the following command: ``export HOMEBREW_GITHUB_API_TOKEN=your_token_here``
+ - [ ] Run the following command to automatically update the formula and create a PR in the Homebrew repository: ``brew bump-formula-pr --version=$VERSION solidity``
+ Note: the version should not include the prefix ``v``, e.g. ``X.Y.Z``.
```
Comment (cameel) [inline ReleaseChecklist.md:92]:
```suggestion
 - [ ] Run the following command to automatically update the formula and create a PR in the Homebrew repository: ``brew bump-formula-pr --version=$VERSION solidity``.
     Note: the version should not include the ``v`` prefix, i.e. ``X.Y.Z``.
```

File: ReleaseChecklist.md [hunk 6]
```diff
@@ -103,6 +116,7 @@ At least a day before the release:
  - [ ] Wait for the CI runs on the tag itself.
  - [ ] Take the ``solc-x.y.z.tgz`` artifact from ``build-package`` run on the tagged commit in circle-ci.
        Inspect the tarball to ensure that it contains an up-to-date compiler binary (``soljson.js``).
+       See the section "Testing version of soljson.js": https://github.com/ethereum/solidity/wiki/Building-solc%E2%80%90js-from-source#testing-version-of-soljsonjs
```
Comment (cameel) [inline ReleaseChecklist.md:119]:
```suggestion
       See the [Testing version of soljson.js](https://github.com/ethereum/solidity/wiki/Building-solc‐js-from-source#testing-version-of-soljsonjs) section on the wiki.
```

**Review by cameel — COMMENTED** [review]
Needs some formatting corrections.

**Pull request closed without merging**